CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Internal: Migrated `QRCode` to typescript
 - Internal: Migrated `PhoneNumberInput` to typescript
 - Internal: Migrated `locales/polyglot` to typescript
+- Public: Fix issue where `poa` and `document` steps cannot be used at the same time, due to steps overwriting data from one another. Adds `poa` key to `onComplete` callback, when `poa` step is used.
```

Comment:
This would be part of a new SDK version, no?

Reply:
Yep. I've made the change locally, but cannot push currently due to github issues.

Reply:
Now corrected.